interoperator/pkg/constants/constants.go

```diff
@@ -7,6 +7,7 @@ import (
 // Constants used by interoperator
 const (
 	FinalizerName    = "interoperator.servicefabrik.io"
+	BrokerFinalizer  = "broker.servicefabrik.io"
```

Comment:
This is only being used in tests. You can hardcode this string in the tests IMO. Once you put something in constants people may start abusing it. Ideally interoperator should have nothing to do with broker finalizer.

Reply:
Done